pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -931,6 +931,28 @@ func (m *kubeGenericRuntimeManager) computeInitContainerActions(pod *v1.Pod, pod
 			}
 
 			if types.IsRestartableInitContainer(container) {
```

Comment:
we will also need to figure out whether we need to call `internalLifecycle.PostStopContainer` for restartable init contianers. I think it is needed for topology manager. We may do it as a separate PR, just need to track it somehow

Reply:
I guess it is already called by `removeContainer`.